app/services/cfe_civil/components/properties.rb

```diff
@@ -33,7 +33,7 @@ def second_home
       end
 
       def or_zero(number)
```

Comment:
Looks like this method could be used elsewhere in this file (or removed entirely because it's not doing very much) but not a blocker

Reply:
Good spot... theses were auto fixes and I'd like to investigate that a bit more 🤔

Reply:
@agoldstone93 I ended up removing it and refactoring the calling methods, are you okay with this?

Reply:
Yep. Fine with me

Is the `|| 0.0` necessary? The fact that it was removed automatically from the method got me thinking. I would assume that if the left side is nil, it would become 0.0 regardless (because `nil.to_s` is `0.0`). Are there any other scenarios it's covering? This isn't a major issue though; I don't think it's worth spending time looking into if you don't know off the top of your head

Reply:
It's weird that rubocop didn't like it in isolation but is happy with it in this form! 🤔 
It's possible it could be removed... if you are looking for tickets @agoldstone93 feel free to have a play :+1: